docs/content/learn/tutorials/control/basic-profile.md

```diff
@@ -0,0 +1,1280 @@
+
+Now that we've seen a basic example, lets take a quick walkthrough of some of the basic functions of this section and how to use them.
+
+### Including Controls from a Catalog
+
+Each `import` object contains directives on which catalog will be included and present in the outcome.
+
+Controls can be included based on pattern matching:
+
+{{< tabs XML JSON YAML >}}
+{{% tab %}}
+
+```xml {linenos=table}
+<import href="https://raw.githubusercontent.com/usnistgov/oscal-content/main/examples/catalog/json/basic-catalog.json">
+    <include-controls>
+      <matching pattern="s1.1.*"/>
+    </include-controls>
+</import>
+```
+
+{{% /tab %}}
+{{% tab %}}
+```json {linenos=table}
+"imports": [
+      {
+        "href": "https://raw.githubusercontent.com/usnistgov/oscal-content/main/examples/catalog/json/basic-catalog.json",
+        "include-controls": [
+          {
+            "matching": [
+              {
+                "pattern": "s1.1.*"
+              }
+            ]
+          },
+```
+
+{{% /tab %}}
+{{% tab %}}
+```yaml {linenos=table}
+  imports:
+    - href: https://raw.githubusercontent.com/usnistgov/oscal-content/main/examples/catalog/json/basic-catalog.json
+      include-controls:
+        - matching:
+            - pattern: s1.1.*
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Controls can also be included based on id matching:
+
+{{< tabs XML JSON YAML >}}
+{{% tab %}}
+
+```xml {linenos=table}
+<import href="https://raw.githubusercontent.com/usnistgov/oscal-content/main/examples/catalog/json/basic-catalog.json">
+    <include-controls>
+      <with-id>s2.1.2</with-id>
+    </include-controls>
+</import>
+```
+
+{{% /tab %}}
+{{% tab %}}
+```json {linenos=table}
+ "imports": [
+      {
+        "href": "https://raw.githubusercontent.com/usnistgov/oscal-content/main/examples/catalog/json/basic-catalog.json",
+        "include-controls": [
+          {
+            "with-ids": [
+              "s2.1.2"
+            ]
+          }
+        ]
+      }
+    ],
+
+```
+
+
+{{% /tab %}}
+{{% tab %}}
+```yaml {linenos=table}
+imports:
+    - href: https://raw.githubusercontent.com/usnistgov/oscal-content/main/examples/catalog/json/basic-catalog.json
+      include-controls:
+        - with-ids:
+            - s2.1.2
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Excluding Controls from a Catalog
+
+It is possible to exclude controls from a catalog. Exclusions work the same way as inclusions; except in this case the indicated control(s) do  not appear in the target catalog.
+The OSCAL profile Resolution Specification Draft contains a more detailed explanation of [excluding controls](https://pages.nist.gov/OSCAL/concepts/processing/profile-resolution/#d2e589-head).
+
+{{< tabs XML JSON YAML >}}
+{{% tab %}}
+
+```xml {linenos=table}
+<import href="https://raw.githubusercontent.com/usnistgov/oscal-content/main/examples/catalog/json/basic-catalog.json">
+    <exclude-controls>
```

Comment:
We decided against having code examples that weren't part of the final example to avoid confusing the reader. Unfortunately excluding is not really possible to do coherently with the basic catalog. Michaela suggested we do a second tutorial focused on advanced profile topics.

Reply:
I am going to update with a recommendation here in a little while, but I do not see why with the aforementioned catalog we cannot include `s2.1.*` and exclude `s2.1.2` as the catalog has more or less two controls from a brief skim. We have to do that or remove this section altogether, as I am not sure how this is more confusing.